src/components/post/PostView.jsx

```diff
@@ -34,7 +34,7 @@ class PostView extends Component {
         const newPost = {};
         const post = response.posts[i];
 
-        if (!post.unlisted && !post.content_type.includes("image")) {
+        if (!post.unlisted && (!post.content_type || !post.content_type.includes("image"))) {
```

Comment:
@zhaoyuxuan can we make these checks in the backend instead of here? Since the image posts never need to be rendered so there is no point of sending them. And also, the user should be able to see their own unlisted posts - we can display some icon for unlisted posts.